docs/ref/checks.txt

```diff
@@ -343,11 +343,11 @@ Models
 * **models.E028**: ``db_table`` ``<db_table>`` is used by multiple models:
   ``<model list>``.
 * **models.E029**: index name ``<index>`` is not unique for model ``<model>``.
-* **models.E030**: index name ``<index>`` is not unique amongst models:
+* **models.E030**: index name ``<index>`` is not unique among models:
   ``<model list>``.
 * **models.E031**: constraint name ``<constraint>`` is not unique for model
   ``<model>``.
-* **models.E032**: constraint name ``<constraint>`` is not unique amongst
+* **models.E032**: constraint name ``<constraint>`` is not unique among
```

Comment:
These checks messages changes will also need making in the code. Please can you do a pass checking the whole codebase, not just the docs? There are many user-facing strings in the code.

Reply:
Thanks Adam. I've reviewed the codebase and reflected the proposed changes there as well.